components/admin_console/system_users/system_users_dropdown/__snapshots__/system_users_dropdown.test.tsx.snap

```diff
@@ -67,7 +67,7 @@ exports[`components/admin_console/system_users/system_users_dropdown/system_user
       </li>
       <li>
         <Memo(MemoizedFormattedMessage)
-          defaultMessage="Bot accounts they manage will be disabled along with their integrations. To enable them again, go to <linkBots>Integrations > Bot Accounts</link>. <linkDocumentation>Learn more about bot accounts</linkDocumentation>. \\\\n \\\\n \\\\n"
+          defaultMessage="Bot accounts they manage will be disabled along with their integrations. To enable them again, go to <linkBots>Integrations > Bot Accounts</link>. <linkDocumentation>Learn more about bot accounts</linkDocumentation>.\\\\n \\\\n \\\\n"
```

Comment:
Thank you for catching up my comments @ctlaltdieliet .

But I found an additional inconsistency about XML tag in this message. (I wish I could have found this before...) 

```suggestion
          defaultMessage="Bot accounts they manage will be disabled along with their integrations. To enable them again, go to <linkBots>Integrations > Bot Accounts</linkBots>. <linkDocumentation>Learn more about bot accounts</linkDocumentation>.\\\\n \\\\n \\\\n"
```

Should I create another PR for this? @amyblais @M-ZubairAhmed

Reply:
I fixed the xml-tag and maybe it can merged again?